### PR TITLE
Reviewer can complete 8

### PR DESCRIPTION
**Pull request by shakalee14**
- Added check for collaborator on repo before a user can create a PRRR

### CODE REVIEW

**Comment by shakalee14** [comment]
@deadlyicon i don't know what happened to the PR but the shah that addresses that bug is 7b3e2c2

**Comment by shakalee14** [comment]
@deadlyicon

File: server/queries/index.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ export default class Queries {
       .select('*')
       .from('pull_request_review_requests')
       .orderBy('created_at', 'asc')
+      .where('completed_at', null )
```
Comment (deadlyicon) [inline server/queries/index.js]: ` .where('completed_at', null )` -> `.where('completed_at', null)`

File: server/database/migrations/20170104083933_add_completed_column.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+exports.up = (knex, Promise) => {
```
Comment (deadlyicon) [inline server/database/migrations/20170104083933_add_completed_column.js]: please follow the coding style in the other migrations

File: server/commands/index.js [hunk 3]
```diff
@@ -153,9 +171,24 @@ export default class Commands {
...
+      completed_at: new Date,
+    })
+    .where('id', prrrId)
+    .returning('*')
+    .then(firstRecord)
+  }
+
+  completePrrr(prrrId){
+    return this.markPullRequestAsComplete(prrrId)
+  }
```
Comment (deadlyicon) [inline server/commands/index.js]: lets just have the one method
```js
completePrrr(prrrId){
  return this.knex
  .table('pull_request_review_requests')
  .update({
    completed_at: new Date,
  })
  .where('id', prrrId)
  .returning('*')
  .then(firstRecord)
}
```

File: server/commands/index.js [hunk 4]
```diff
@@ -60,6 +60,23 @@ export default class Commands {
         error.status = 400
         throw error
       })
+      .then(pullRequest => {
```
Comment (deadlyicon) [inline server/commands/index.js]: remove this from this PR. it was already merged

File: server/api/index.js [hunk 5]
```diff
@@ -70,6 +70,15 @@ router.post('/pull-request-review-requests/:prrrId/destroy', (req, res, next) =>
     .catch(next)
 });
 
+router.post('/pull-request-review-requests/:prrrId/complete', (req, res, next) => {
+  const { prrrId } = req.params
+  req.commands.completePrrr(prrrId)
+    .then(_ => {
+      res.json()
```
Comment (deadlyicon) [inline server/api/index.js]: `res.json()` -> `res.json({id: prrrId, complete: true})`

File: browser/components/molecules/CompletedPrrrs/index.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+      <td key="completed">
+        <span>by&nbsp;</span>
+        <GithubUsername username={prrr.claimed_by} currentUser={currentUser} />
+        <span>&nbsp;</span>
+        <Date fromNow date={prrr.completed_at} />
+      </td>
+    ]
+  }
+
+  renderCompletedPullRequests = (prrr) => {
```
Comment (deadlyicon) [inline browser/components/molecules/CompletedPrrrs/index.js]: this doesnt look like it does anything. do we need it?

File: browser/components/molecules/CompletedPrrrs/index.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
+import React, { Component, PropTypes } from 'react'
```
Comment (deadlyicon) [inline browser/components/molecules/CompletedPrrrs/index.js]: please remove `CompletedPrrrs` for this PR. we'll add it back later

File: browser/components/molecules/ClaimedPrrrs/index.js [hunk 8]
```diff
@@ -6,17 +6,30 @@ import Button from '../../atoms/Button'
...
+import completePrrr from '../../../actions/completePrrr'
+
 
 export default class ClaimedPrrrs extends Component {
   static propTypes = {
     currentUser: PropTypes.object.isRequired,
     prrrs: PropTypes.array.isRequired,
   }
 
+  completePrrr(prrr){
```
Comment (deadlyicon) [inline browser/components/molecules/ClaimedPrrrs/index.js]: remove this class method and lets just call `completePrrr` directly.

File: browser/components/molecules/ClaimedPrrrs/index.js [hunk 9]
```diff
@@ -6,17 +6,30 @@ import Button from '../../atoms/Button'
...
+    console.log('what is undefined', prrr)
+    completePrrr(prrr.id)
+    .then(_ => {
```
Comment (deadlyicon) [inline browser/components/molecules/ClaimedPrrrs/index.js]: we also dont need to open the PR in another tab when we complete it.

File: README.md [hunk 10]
```diff
@@ -5,13 +5,48 @@
 ![prrr](http://www.kittenswhiskers.com/wp-content/uploads/sites/48/2014/05/cat-purring.jpg)
```
Comment (deadlyicon) [inline README.md]: remove all the README changes please

**Pull request merged**